src/luascript.cpp

```diff
@@ -2553,10 +2557,56 @@ void LuaScriptInterface::registerFunctions()
 
 	registerMethod("Condition", "addDamage", LuaScriptInterface::luaConditionAddDamage);
 
+	// NewMonsterType
+	registerClass("NewMonsterType", "", LuaScriptInterface::luaNewMonsterTypeCreate);
+
 	// MonsterType
 	registerClass("MonsterType", "", LuaScriptInterface::luaMonsterTypeCreate);
```

Comment:
Maybe it could be done in the way that tile related functions are done?
For example:
```lua
MonsterType(name) -- returns existing MonsterType or nil if it doesn't exist
Game.createMonsterType(name) -- returns existing MonsterType or creates and returns new if it doesn't exist
```